src/stories/pages/Map/Map.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { MapContainer, TileLayer, Marker, Popup } from "react-leaflet";
+import "leaflet/dist/leaflet.css";
+import styles from "./Map.module.css";
+
+export default function Map() {
+  return (
+    <MapContainer
+      className={styles.map}
+      center={[51.505, -0.09]}
+      zoom={13}
+      scrollWheelZoom={false}
+    >
+      <TileLayer url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png" />
+      <Marker position={[51.505, -0.09]}>
+        <Popup>
+          Visit Rosenheim <br />
+        </Popup>
```

Comment:
Something doesn't seem to work. Maybe the Marker image?
![image](https://user-images.githubusercontent.com/48555130/144413659-71f406fb-11e3-4b26-866e-d2943d2b1d0d.png)

Reply:
Hi, that's a integrated popup where i put 'Visit Rosenheim' inside for testing. I'll check it out 👍